_posts/2016/2016-10-12-using-bower-with-yarn.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Using Bower with Yarn"
+author: benmann
+---
+
+Yesterday's official release of [Yarn](https://code.facebook.com/posts/1840075619545360) adds an interesting new
+tool to the world of package managers, bridging the gap between Bower and npm. It looks like a great improvement compared
+to the current Bower client in a number of ways and also a possible option for transitioning from Bower to npm, if that
+is something you've been looking for recently.
+
+As a long time Bower user, transitioning to Yarn should be a pleasent experience, as it is supposed to be a drop-in alternative
+to the Bower client. That means you can continue using your existing `bower.json` just like before.
+Here's a quick look at some more goodies Yarn has to offer in comparison to the current Bower client.
+
+### Lockfile
+Faithful users of Bower will be very pleased to see that Yarn adds the long awaited lockfile feature with even stricter
+reproducability across devices and systems than npm offer at the moment.
+
+### Speed and offline
+Yarn's cache is a pretty elaborate improvement over what Bower had to offer so far, reducing (cold) install times by a great
+deal and even more so in a chached scenario. At the same time it allows complete installs even without an internet connection!
+As a bonus Yarn will retry network requests during an installation in case a single dependency couldn't be resolved right away.
+
+### Security
+Checking each packages checksum before they are executed guarantees a certain degree of integrity and gives
+the developer more confidence from a security standpoint.
+
+### Licenses
+Being able to see every dependencies License is an often needed requirement in corporate scenarios.
```

Comment:
The ability to list a dependencies' license type is often required in enterprise environments. Yarn offers an easy to list the license type for a given dependency.